### PR TITLE
Roadmap update

### PR DESCRIPTION
**Pull request by bame-da**
### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [x] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/roadmap.rst [hunk 1]
```diff
@@ -1,42 +1,45 @@
...
 
-  You can read about the release stages on the :doc:`/daml-integration-kit/index` page. 
+  Public release of a pre-alpha reference distributed DAML Ledger implementation with a public test-net, strong privacy, regulatory compliance, and composabilty
+
+  See https://canton.io/
+
+- **Deployment Options**
+
+  - DAML-on-Aurora publicly available on AWS Marketplace
+  - DAML-on-Sawtooth publicly available on Sextant by Blockchain Technology Partners (https://blockchaintp.com/sextant/)
```
Comment (meiersi-da) [inline docs/source/support/roadmap.rst]: use proper link?

File: docs/source/support/roadmap.rst [hunk 2]
```diff
@@ -1,42 +1,45 @@
...
-This page specifies the major features we’re planning to add next to the DAML SDK. Plans and timelines are subject to change. If you need any of these features or want to request others, see the :doc:`/support/support` page for how to get in touch.
+This page specifies the major features we're planning to add next to the DAML Ecosystem. Plans and timelines are subject to change. If you need any of these features or want to request others, see the :doc:`/support/support` page for how to get in touch.
 
 We plan to update this roadmap roughly every three months.
 
-- **Generic templates**
+- **DAML Triggers**
 
-  Support for writing generic templates - for example, a generic ``Proposal`` workflow that could be instantiated as a ``Proposal Iou`` to propose an ``Iou``.
+  Support for non-transactional automation written directly in DAML. See `discusison on GitHub <https://github.com/digital-asset/daml/issues/1615>`_
```
Comment (gerolf-da) [inline docs/source/support/roadmap.rst]:
```suggestion
  Support for non-transactional automation written directly in DAML. See `discussion on GitHub <https://github.com/digital-asset/daml/issues/1615>`_
```

File: docs/source/support/roadmap.rst [hunk 3]
```diff
@@ -1,42 +1,45 @@
 .. Copyright (c) 2019 The DAML Authors. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-DAML SDK roadmap (as of June 2019)
-==================================
+DAML roadmap (as of September 2019)
+===================================
 
-This page specifies the major features we’re planning to add next to the DAML SDK. Plans and timelines are subject to change. If you need any of these features or want to request others, see the :doc:`/support/support` page for how to get in touch.
+This page specifies the major features we're planning to add next to the DAML Ecosystem. Plans and timelines are subject to change. If you need any of these features or want to request others, see the :doc:`/support/support` page for how to get in touch.
```
Comment (gerolf-da) [inline docs/source/support/roadmap.rst]:
```suggestion
This page specifies the major features we are planning to add next to the DAML Ecosystem. Plans and timelines are subject to change. If you need any of these features or want to request others, see the :doc:`/support/support` page for how to get in touch.
```

File: docs/source/support/roadmap.rst [hunk 4]
```diff
@@ -1,42 +1,45 @@
...
 
-  Tools that allow users to visualise the static workflow and lifecycle of contracts.
-- **Improve app portability between DAML ledgers**
+  - Package Management
+  - Party Allocation and Management
+  - Application Deployment
+  - Application Upgrading
 
-  Add a DAML-on-X certification tool that tests that DAML ledgers implement all DAML functionality, and document a portable approach to DAML application testing.
+- **API Authentication**
```
Comment (gerolf-da) [inline docs/source/support/roadmap.rst]:
```suggestion
- **API Authentication and Authorization**
```

File: docs/source/support/roadmap.rst [hunk 5]
```diff
@@ -1,42 +1,45 @@
...
 
-  `Read more on GitHub <https://github.com/digital-asset/TODO>`__.
-- **Ledger API authentication**
+  Addition of JWT based authentication to the Ledger API
```
Comment (gerolf-da) [inline docs/source/support/roadmap.rst]:
```suggestion
  Addition of JWT based authentication and authorization to the Ledger API
```

**Comment by hurryabit** [comment]
The parts for the language team look good.

**Pull request merged**